src/containers/ProjectContentsContainer.jsx

```diff
@@ -125,7 +131,6 @@ class ProjectContentsContainer extends Component {
           onChange={this.handleChange}
           placeholder="Translate some text"
           type="text"
-          value={this.state.translationText}
         />
```

Comment:
This isn't setting a value for the input field, even if there is text in the translation.